### PR TITLE
[IMP] accounting: intrastat

### PR DESCRIPTION
**Pull request by toaa-odoo**
Updated Intrastat for 15.0

task-3373480

### CODE REVIEW

File: content/applications/finance/accounting/reporting/intrastat.rst [hunk 1]
```diff
@@ -2,27 +2,173 @@
...
+To add the product's country of origin, go to :menuselection:`Accounting --> Customers --> Products`
+and select a product. Under the :guilabel:`Accounting` tab, set the :guilabel:`Country of Origin`.
+
+.. _intrastat/invoice-bill-configuration:
+
+Invoices and bills configuration
+================================
+
+Once products are properly configured, several settings must be configured on the invoices and bills
+you create.
```
Comment (chiaraprattico) [inline content/applications/finance/accounting/reporting/intrastat.rst:92]:
```suggestion
Once products are properly configured, several settings must be enabled on the invoices and bills
you create.
```

File: content/applications/finance/accounting/reporting/intrastat.rst [hunk 2]
```diff
@@ -2,27 +2,173 @@
...
+.. seealso::
+   `National Bank of Belgium - Intrastat: Nature of transactions from January 2022
+   <https://www.nbb.be/doc/dd/onegate/data/new_natures_of_transaction_2022_en.pdf>`_
+
+.. _intrastat/partner-country:
+
+Partner country
+---------------
+
+The partner country represents the vendor's country in case of bills, and the customer's country in
```
Comment (chiaraprattico) [inline content/applications/finance/accounting/reporting/intrastat.rst:116]: I'd slightly change this to make it clearer

File: content/applications/finance/accounting/reporting/intrastat.rst [hunk 3]
```diff
@@ -2,27 +2,173 @@
...
+case of invoices. It is automatically filled in using the :guilabel:`Country` field in the **contact
+form card** of the partner.
```
Comment (chiaraprattico) [inline content/applications/finance/accounting/reporting/intrastat.rst:118]: Isn't it just contact form? without card

File: content/applications/finance/accounting/reporting/intrastat.rst [hunk 4]
```diff
@@ -2,27 +2,173 @@
...
+
+To edit the partner country manually, when on an invoice or bill, click the :guilabel:`Other Info`
+tab, and select the :guilabel:`Intrastat Country`.
```
Comment (chiaraprattico) [inline content/applications/finance/accounting/reporting/intrastat.rst:121]:
```suggestion
To edit the partner country manually, click on the :guilabel:`Other Info`
tab on an invoice or bill, and select the :guilabel:`Intrastat Country`.
```

File: content/applications/finance/accounting/reporting/intrastat.rst [hunk 5]
```diff
@@ -2,27 +2,173 @@
...
+
+.. _intrastat/transport-code:
+
+Transport code
+~~~~~~~~~~~~~~
+
+The transport code identifies the presumed **mode of transport** used to send the goods (arrival or
+dispatch). Once you have selected a :ref:`partner country <intrastat/partner-country>`, the
+:guilabel:`Intrastat Transport Mode` becomes available underneath it.
```
Comment (chiaraprattico) [inline content/applications/finance/accounting/reporting/intrastat.rst:130]: just becomes available, without underneath it .. ?

File: content/applications/finance/accounting/reporting/intrastat.rst [hunk 6]
```diff
@@ -2,27 +2,173 @@
...
+
+.. _intrastat/value:
+
+Value of the goods
+------------------
```
Comment (chiaraprattico) [inline content/applications/finance/accounting/reporting/intrastat.rst:135]: change h level (?)

File: content/applications/finance/accounting/reporting/intrastat.rst [hunk 7]
```diff
@@ -2,27 +2,173 @@
...
+
+The value of a good is the untaxed :guilabel:`Subtotal` (:guilabel:`Price` multiplied by
+:guilabel:`Quantity`) of an invoice line.
+
+.. _intrastat/partner:
+
+Partner configuration
+=====================
+
+Two fields from the partner's **contact form card** are used with Intrastat: :guilabel:`VAT` and
```
Comment (chiaraprattico) [inline content/applications/finance/accounting/reporting/intrastat.rst:145]: contact form (card)

File: content/applications/finance/accounting/reporting/intrastat.rst [hunk 8]
```diff
@@ -2,27 +2,173 @@
...
+invoice or bill.
+
+Generate the Intrastat report
+=============================
+
+Generate the report by going to :menuselection:`Accounting --> Reporting --> Audit Reports:
+Intrastat Report`. It is automatically computed based on the :ref:`default configuration
+<intrastat/general-configuration>` and the information found on the :ref:`products
+<intrastat/product-configuration>`, :ref:`invoices and bills
+<intrastat/invoice-bill-configuration>`, and :ref:`partners <intrastat/partner>`.
```
Comment (chiaraprattico) [inline content/applications/finance/accounting/reporting/intrastat.rst:156]:
```suggestion
Generate the report by going to :menuselection:`Accounting --> Reporting --> Audit Reports:
Intrastat Report`. It is automatically computed based on the :ref:`default configuration
<intrastat/general-configuration>` and the information on the :ref:`products
<intrastat/product-configuration>`, :ref:`invoices and bills
<intrastat/invoice-bill-configuration>`, and :ref:`partners <intrastat/partner>`.
```

File: content/applications/finance/accounting/reporting/intrastat.rst [hunk 9]
```diff
@@ -2,27 +2,173 @@
...
-*Save*. Once the Intrastat has been activated you are able to choose the **Default incoterm**.
+Each report line refers to a single invoice line and contains the following information:
 
-.. important::
-   If you created your Odoo database prior to **January 13, 2022**, please :ref:`upgrade
-   <general/upgrade>` the ``account_intrastat`` module to add the new transaction codes and
-   :ref:`install <general/install>` the ``account_intrastat_expiry`` module to archive the old
-   codes.
+- Invoice or bill reference number;
+- System, which is a code automatically generated depending on whether the document is an invoice
```
Comment (chiaraprattico) [inline content/applications/finance/accounting/reporting/intrastat.rst:163]: :guilabel:'System'

**Review by chiaraprattico — CHANGES_REQUESTED** [review]

File: content/applications/finance/accounting/reporting/intrastat.rst (hunk 6 above)
Comment (toaa-odoo) [reply content/applications/finance/accounting/reporting/intrastat.rst:135]: I still think it makes more sense as such

**Review by chiaraprattico — APPROVED** [review]

**Comment by xpl-odoo** [comment]
@toaa-odoo The Intrastat report on 15.0 contains the same columns as on 16.0, yet you don't mention:
- Incoterm Code
- Weight
- Supplementary Units
I think @jcs-odoo moved the info about Incoterms to another page, as they don't only apply to Intrastat. But it's not clear on the current Intrastat (16.0+) pages. So it should be updated IMO. 
Regarding weight and supplementary units, it's probably taken from another product tab.
Could you investigate? :)

**Comment by toaa-odoo** [comment]
> @toaa-odoo The Intrastat report on 15.0 contains the same columns as on 16.0, yet you don't mention:
> 
> * Incoterm Code
> * Weight
> * Supplementary Units
> 
> I think @jcs-odoo moved the info about Incoterms to another page, as they don't only apply to Intrastat. But it's not clear on the current Intrastat (16.0+) pages. So it should be updated IMO. Regarding Weight and Supplementary Units, it's probably taken from another product tab. Could you investigate? :)

Hi XPL, it's as I thought: it's not implemented in V15.
benj

**Review by xpl-odoo — APPROVED** [review]
:)

**Comment by xpl-odoo** [comment]
@fw-bot up to saas-15.2

**Comment by xpl-odoo** [comment]
@robodoo r+

**Pull request closed without merging**